literals/README.md

```diff
@@ -0,0 +1,13 @@
+## Literals
+The directory contains all the literals that are used in emails sent out. The `LiteralType` contains a type with `subject`, `body` and `greeting` (optional) as needed for all the emails. All the `literal` Objects defined to handle the different possible responses are as listed below.
```

Comment:
I have trouble understanding this paragraph, especially the last sentence. Is there a typo? What are "`literal` Objects"?

Reply:
Let me change it to "`literal` Object is defined to handle as listed below."

Reply:
Why is "Object" capitalized? Actually, could you commit the supposed change?